ro_help/hub/views.py

```diff
@@ -199,7 +199,7 @@ def get_context_data(self, **kwargs):
         context["current_search"] = self.request.GET.get("q", "")
         context["current_tags"] = self.request.GET.getlist("tag", "")
         context["counties"] = needs.order_by("county").values_list("county", flat=True).distinct("county")
-        context["tags"] = sorted(set(needs.values_list("resource_tags__name", flat=True)))
+        context["tags"] = sorted(set([n for n in needs.values_list("resource_tags__name", flat=True) if n is not None]))
```

Comment:
can we change this to `if not n`?

Do you see a reason why n would ever be valid as a falsey value?